hw05-framework/src/main/java/com/remifo/TestFramework.java

```diff
@@ -0,0 +1,88 @@
+package com.remifo;
+
+import com.remifo.annotations.After;
+import com.remifo.annotations.Before;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TestFramework {
+
+    private int numberSuccessfulTest;
+    private int numberUnsuccessfulTest;
+
+    public void startTest( String name ) {
+        Method beforeMethod = null;
+        Method afterMethod = null;
+        Set<Method> testMethods = new HashSet<>();
+
+        Method[] methods = getMethods( name );
+
+        for ( Method method : methods ) {
+            method.setAccessible( true );
+            if ( method.isAnnotationPresent( Before.class ) )
+                beforeMethod = method;
+            else if ( method.isAnnotationPresent( After.class ) ) {
+                afterMethod = method;
+            } else {
+                testMethods.add( method );
+            }
+        }
+
+        for ( Method method : testMethods ) {
+            Object object = getConstructor(name);
```

Comment:
Здесь же вы получаете не конструктор, а уже новый объект.
```suggestion
            Object object = createObject(name);
```